src/ui/public/kbn_top_nav/kbn_top_nav_controller.js

```diff
@@ -53,13 +53,18 @@ export default function ($compile) {
 
     // apply the defaults to individual options
     _applyOptDefault(opt = {}) {
-      return defaults({}, opt, {
+      const defaultedOpt = defaults({}, opt, {
```

Comment:
I think this can be rewritten without lodash:

```javascript

      const defaultedOpt = Object.assign({
        label: capitalize(opt.key),
        hasFunction: !!opt.run,
        description: opt.run ? opt.key : `Toggle ${opt.key} view`,
        run: (item) => !item.disableButton && this.toggle(item.key)
      }, opt);
```